tools/runners/run-mongo.js

```diff
@@ -213,8 +213,7 @@ if (process.platform === 'win32') {
             var foundPort = parseInt(m[2], 10);
             var foundPath = m[3];
 
-            if ( (! port || port === foundPort) &&
-                 (! dbDir || dbDir === foundPath)) {
+            if (! dbDir || dbDir === foundPath) {
```

Comment:
This seems incorrect. `findMongoPids` is used in two places:

1. [To find the mongo port](https://github.com/meteor/meteor/blob/b484edf66964ad5e90dd8c2b9fc38653e3bad7f5/tools/runners/run-mongo.js#L235) as `findMongoPids(dbDir)`

2. [To kill the running mongo](https://github.com/meteor/meteor/blob/b484edf66964ad5e90dd8c2b9fc38653e3bad7f5/tools/runners/run-mongo.js#L297) as `findMongoPids(null, port)`.

In the first case, this change won't do anything as `port` is undefined. In the second case, this will now match *all* mongo processes, as `dbDir` is `null`.

I don't understand the reason for this change I guess?